lich.rbw

```diff
@@ -8736,20 +8758,40 @@ module Effects
       @dialog = dialog
     end
 
-    def to_h
-      XMLData.dialogs.fetch(@dialog, {})
+    def to_h(complex: false)
```

Comment:
this should not be done, it should be the same as before

Reply:
I may support this view based on the offered simplifications - presuming I get them in and can make them work to both existing and future needs.  However, at present, this feature does not break anything.  It may require modification / extension depending on the open question of best `active_spell` display methods, but can be changed anytime including after PR merge. I would appreciate your review of this commend and cocurrence, then mark the discussion resolved purely in the context of 'should be the same' given the stated design goal.  See exchange in Lich Dev server for detail.

Reply:
I have opted to extend the `to_h` def as shown, but with a better understanding of what you are saying here, O, I have reverted the `to_h` non extended to be specific only to the data in parse_psm3_progressbar.  As long as the extension concept passes muster, this also is resolved.